app/models/service_request.rb

```diff
@@ -566,6 +560,16 @@ def is_editable?
     return true
   end
 
+  def has_locked_ssr?
+    sub_service_requests.each do |ssr|
+      if !ssr.can_be_edited?
+        return true
```

Comment:
I tried removing the returns, and the functionality did not work the same way. The return appears to be needed